src/vis/scatter/interfaces.ts

```diff
@@ -9,8 +9,15 @@ export interface IScatterConfig extends BaseVisConfig {
   dragMode: EScatterSelectSettings;
   alphaSliderVal: number;
   sizeSliderVal: number;
+  showLabels: ELabelingOptions;
 }
 
 export function isScatterConfig(s: BaseVisConfig): s is IScatterConfig {
   return s.type === ESupportedPlotlyVis.SCATTER;
 }
+
+export enum ELabelingOptions {
+  SELECTED = 'Selected',
+  ON = 'On',
+  OFF = 'Off',
```

Comment:
I suggest to adapt the label and enums to:

```suggestion
  NEVER = 'Never',
  ALWAYS = 'Always',
  SELECTED = 'Selected',
```